src/thunderbots/software/ai/hl/stp/tactic/free_kick_tactic.cpp

```diff
@@ -0,0 +1,96 @@
+#include "free_kick_tactic.h"
+
+#include "ai/hl/stp/action/chip_action.h"
+#include "ai/hl/stp/action/kick_action.h"
+#include "ai/hl/stp/evaluation/calc_best_shot.h"
+#include "ai/hl/stp/evaluation/deflect_off_enemy_target.h"
+#include "ai/hl/stp/evaluation/indirect_chip.h"
+#include "shared/constants.h"
+#include "util/logger/init.h"
+
+namespace
+{
+    /**
+     * Minimum opening seen by robot for a shot on goal.
+     */
+    const Angle MIN_SHOT_ANGLE = Angle::ofDegrees(15);
```

Comment:
We could probably shoot with an opening of 6 degrees? That would allow for an error of 3 degrees in either direction, which we should be able to be within.

Reply:
changed